src/domain/Lotto.js

```diff
@@ -0,0 +1,44 @@
+import ERROR_MESSAGE from '../constants/errorMessage.js';
+import CustomError from '../CustomError.js';
+import LottoNumberValidator from '../LottoNumberValidator.js';
+
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.#validate(numbers);
+    this.#numbers = this.#sortNumbers(numbers);
+  }
+
+  #validate(numbers) {
+    this.#validateNumbersLength(numbers);
+    this.#validateNumbersDuplicate(numbers);
+
+    numbers.forEach((number) => {
+      LottoNumberValidator.validateLottoNumber(number);
+    });
+  }
+
+  #validateNumbersLength(numbers) {
+    if (numbers.length !== 6) {
+      throw new CustomError(ERROR_MESSAGE.NOT_SIX_LENGTH);
+    }
+  }
+
+  #validateNumbersDuplicate(numbers) {
+    if (new Set(numbers).size !== numbers.length) {
+      throw new CustomError(ERROR_MESSAGE.DUPLICATE_NUMBER);
+    }
+  }
+
+  #sortNumbers(numbers) {
+    const translatedNumbers = numbers.map(Number);
+    return translatedNumbers.sort((firstNumber, secondNumber) => firstNumber - secondNumber);
+  }
+
+  getNumbers() {
+    return this.#numbers;
+  }
```

Comment:
```suggestion
  get getNumbers() {
    return this.#numbers;
  }
```
getter를 사용하면 동작을 수행하는 메서드가 아니라, 속성처럼 읽히는 계산된 값을 사용한다는게 명시적이어서 가독성이 더 좋아질 것 같아요!

Reply:
오! getter 사용 방식을 예전에 책에서 읽었는데, 실제로 활용할 생각은 못해봤네요.
한 수 배워갑니다! 덕분에 다음 미션에서 getter를 사용해 코드를 작성했어요 🙂👍👍👍